jobs/build/ocp4/Jenkinsfile

```diff
@@ -546,38 +397,43 @@ node {
             }
 
             stage("merge origin") {
+                // note, this will go away once merge is handled in its own job
```

Comment:
It already is. Was merged in yesterday :)

Reply:
Roger, I'll take that out